gatling-jms/src/main/scala/io/gatling/jms/action/JmsReqReply.scala

```diff
@@ -58,7 +59,7 @@ class JmsReqReplyActor(attributes: JmsAttributes, protocol: JmsProtocol, tracker
     extends ValidatedActionActor {
 
   // Create a client to refer to
-  val client = JmsClient(protocol, attributes.destination, attributes.replyDestination)
+  val client: JmsReplyClient = JmsClient(protocol, attributes.destination, attributes.replyDestination.get).asInstanceOf[JmsReplyClient]
```

Comment:
Cf below: move `replyDestination` out of `JmsAttributes` and create 2 different factories.